### DISCUSSION TITLE
Currently visible layout is not distinguished from other layouts when multi-selecting

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#559] [Q&A]
**Description**
- Version: latest web
- OS: macos

**Steps To Reproduce**
Select multiple layouts in the sidebar by cmd-clicking.

**Current Experience**
Since all selected layouts appear identical, it is unclear which layout is **currently** being viewed on the right-hand side of the screen.
<img width="406" alt="image" src="https://user-images.githubusercontent.com/14237/191349968-592bcd93-cc11-43d0-8491-a6917bebcde5.png">

**Desired Experience**
Some indication of which layout is actually being viewed

**Comment by osaro-imohe** [comment]
Quick question:
1) What are those purple dots opposite the layout name for?
2) Can Studio use that to identify the selected layout?

I'm thinking something along the lines of showing the purple dot only for the selected layout @jtbandes

**Comment by jtbandes** [comment]
The purple dot means the layout has been modified since it was last saved.

**Comment by osaro-imohe** [comment]
Ah, I see. What if Studio uses a darker shade of purple as the background color for the selected layout? Thoughts @amacneil @2metres

**Comment by amacneil** [comment]
I'm not convinced we need to solve this problem. The same thing happens in vscode for example. You are only multi-selecting to take an action on those layouts so it doesn't seem critical to know which layout is active?

**Comment by jtbandes** [comment]
VS Code does something a bit different. It distinguishes the most recently clicked file when multi-selecting:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/14237/194470956-169615b1-f895-4440-b130-ab5a368336e5.png">

For VS Code highlighting the currently _open_ file in the editor when multi-selecting isn't a big deal, because the open file is already displayed somewhere, namely in the actual tab:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/14237/194471082-39f60465-d318-4de5-bfad-32929d9ab06b.png">

Plus they have a whole pane dedicated to showing the "open editors".
<img width="322" alt="image" src="https://user-images.githubusercontent.com/14237/194471132-717fb979-4217-4618-b842-3e2b6ec48263.png">

IMO a darker purple as background would be nice, but will let @2metres weigh in
